### ISSUE TITLE
Sets Chart.version to null

### ISSUE THREAD

**Issue by pbatey** [Intility/helm-version#4]
When specifying a specific version like `npm version "1.0.1"` the **Chart.yaml** file ends up with `version: null`.

It should probably increment the Chart.version if the provided version is different from the Chart.appVersion and leave Chart.version unmodified if the provided version matches Chart.appVersion.

**Comment by h3rmanj** [comment]
Hi, thanks for opening this issue ☺️ 

Are you using the latest version? I can't seem to reproduce this. What combination of `appVersion` and `version` fields do you have?

With the current version, the `version` field will be bumped based on the difference between the current `appVersion` and the new version.

**Comment by EwanRoycroft** [comment]
On @intility/helm-version@1.3.0, with `./deploy/Chart.yaml`:
```yaml
# ...

version: 1.0.47

appVersion: "4.11.0"

# ...
```

Running `npx helm-version ./deploy` results in:

```yaml
# ...

version: null

appVersion: "v4.11.0"

# ...
```

However, I cannot reproduce @pbatey's issue using `npm version "4.11.1"`.

**Comment by h3rmanj** [comment]
Thank you for the reproducible variables 😊 

I think I understand what's happening. When running helm-version, the `version` field gets bumped based on the difference in the old and new `appVersion`. I guess since there is no change in `appVersion`, this logic fails and returns null.

I think I'll try to not touch the `version` field when this happens, and leave it to the developer to bump it manually if other changes in the chart has occured.

**Comment by h3rmanj** [comment]
Should be fixed in `1.3.1`, please let me know if it solves your problem ☺️

**Comment by EwanRoycroft** [comment]
Fixed, thanks!
